tests/unit/test_signers.py

```diff
@@ -273,7 +273,7 @@ def test_generate_presigned_post(self):
         self.auth.assert_called_with(
             credentials=self.credentials, region_name='region_name',
             service_name='signing_name')
-        self.add_auth.assert_called_once()
+        self.add_auth.call_count == 1
```

Comment:
Are these changes necessary for this pull request, or are they accidentally included due to the related PR? Can you please remove these changes if they are not necessary.